understanding/20/re-authenticating.html

```diff
@@ -85,94 +80,61 @@ <h2>Examples of Re-authenticating</h2>
                remains intact and, after re-authentication, the user may send that data.</dd>
          <dt>A questionnaire with a time limit</dt>
          <dd>A long questionnaire provided within a single web page has information at the beginning
-               that indicates that the session will time out after 15 minutes. The user is also informed
+               that indicates that the session will time out after 20 hours. The user is also informed
```

Comment:
I feel like there was a possible intention in the original that although this timed out after 15 minutes, the javascript function at the end of this long list of features was intended to allow it to pass due to the "adjust" method (although as written it doesn't).
My feeling is that we've got a LOT of detritus now, when the only thing the author needs to do is set it to time out after 20 hours. The rest is superfluous and risks confusion what is required.
Does it make any sense to either trim it down?

Reply:
In fact, there isn't even a requirement to _inform_ the user of the time out, so just the first sentence would exceed what is necessary.

Reply:
@mbgower right, I remember now ... I chose to change this to 20 hours to align it unequivocally with the timing adjustable SC, as *this* is about the re-authentication SC ... so wanted to focus the thing on the reauthentication aspect, without creating weird doubts about "wouldn't this pass re-authentication but then still fail the timing SC?"

Reply:
On TF call, agreed to strip out this example entirely